docs/examples/viz_parametric_surfaces.py

```diff
@@ -0,0 +1,180 @@
+###############################################################################
+# Importing necessary modules
+
+from fury import window, actor, ui, colormap
+import numpy as np
+
+###############################################################################
+# Function that returns the parametric surface specified by the user.
+
+
+def parametric_func(name, colors=(1, 1, 1), coords=[0, 0, 0],
+                    opacity=1, scale=1.0, cmap=False, cmap_name=None):
+    """
+    Parameters
+    ----------
+    name : string - "mobius_strip" or "kleins_bottle" or "roman_surface"
+        or "boys_surface". Determines the parametric surface to be returned.
+    colors : ndarray (3) or tuple (3)
+        Determines the color of the surface. (default : white)
+    coords : ndarray (3)
+        Determines the coordinates where the parametric surface will be
+        rendered. (default : [0, 0, 0])
+    opacity : float, optional
+        Takes values from 0 (fully transparent) to 1 (opaque). (default : 1)
+    scale : float, optional
+        Determines the size of the parametric surface returned. (default : 1)
+    cmap : bool, optional
+        Set it to true if colormap is being mapped to the surface.
+    cmap_name : string, optional
+        Stores the name of the colormap specified by the user, cmap should be
+        set to True for the colormap to be rendered.
+
+    Returns
+    -------
+    vtkActor
+
+    """
+    def sin(x):
+        return np.sin(x)
+
+    def cos(x):
+        return np.cos(x)
+
+    npoints = 600
+
+    def scaler(arr):
```

Comment:
I think, it would be better to call it normalize.

Reply:
I think this name is ok, `scaler` can be used as a short name for `maxmin_scaler` while `normalization` is more appropriate if the array is being divided by its sum or magnitude.